### PR TITLE
Python plotly bar

### LINKED ISSUE #4557: [Term Entry] Python Plotly .bar()
### Reason/inspiration (optional)

We would like a new term entry in the `Data Visualization` concept for Plotly: `.bar()`. The entry should go in a new file under `docs/content/plotly/concepts/express/terms/bar/bar.md`

The entry should describe the term and how it is used. Also include:

- An introductory paragraph that describes the concept
- A 'Syntax' section that includes the syntax for the concept (if applicable)
- An 'Example' section that shows the concept in use

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by M-MacNeill**
### Description

Term Entry and example output for Plotly .bar() method. Created docs/content/plotly/concepts/express/terms/bar/bar.md and docs\media\plotly-bar-example.png.

### Issue Solved

Closes #4557

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/plotly/concepts/express/terms/bar/bar.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - 'Data Visualization'
+Tags:
+  - 'Data'
+  - 'Graphs'
+  - 'Libraries'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```
Comment (Sriparno08) [inline content/plotly/concepts/express/terms/bar/bar.md:16]:
```suggestion
---
Title: '.bar()'
Description: 'Generates a chart representing categorical data with vertical bars.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Data'
  - 'Graphs'
  - 'Libraries'
  - 'Methods'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

File: content/plotly/concepts/express/terms/bar/bar.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+The **`.bar()`** method in Plotly is part of the Plotly Express high-level interface to Plotly. It generates charts representing categorical data with vertical bars.
```
Comment (Sriparno08) [inline content/plotly/concepts/express/terms/bar/bar.md:18]:
```suggestion
In Plotly, the **`.bar()`** method is part of the high-level interface named `plotly.express`. It generates charts representing categorical data with vertical bars. 
```

File: content/plotly/concepts/express/terms/bar/bar.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+plotly.express.bar(x, y, labels, title, width, height, ...)
+```
+The parameters are:
+
+- x: String, integer, series or array-like designating the x-axis data.
+- y: String, integer, series or array-like designating the y-axis data.
+- labels: Dictionary of string keys and string values designating the labels for the axis titles, etc. 
+- title: String designating the chart title.
+- width: Integer designating the chart width in pixels.
+- height: Integer designating the chart height in pixels.
```
Comment (Sriparno08) [inline content/plotly/concepts/express/terms/bar/bar.md:32]:
````suggestion
```pseudo
plotly.express.bar(x, y, labels, title, width, height, ...)
```

- x: A string, integer, series, or array-like object designating the x-axis data.
- y: A string, integer, series, or array-like object designating the y-axis data.
- labels: A dictionary with string keys and string values designating column names and labels for axis titles, hovers, and legend entries.
- title: A string designating the chart title.
- width: An integer designating the chart width in pixels.
- height: An integer designating the chart height in pixels.
````

File: content/plotly/concepts/express/terms/bar/bar.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+> **Note:** The ellipsis (...) indicates that there can be additional optional parameters beyond those listed here.
+
+## Example
+
+Below is an example demonstrating how to use the `.bar()` method to create a simple bar chart visualizing the values of items in categories a, b, c and d:
```
Comment (Sriparno08) [inline content/plotly/concepts/express/terms/bar/bar.md:38]:
```suggestion
Below is an example demonstrating how to use the `.bar()` method to create a simple bar chart visualizing the values of items in categories `a`, `b`, `c`, and `d`:
```

File: content/plotly/concepts/express/terms/bar/bar.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+```python
```
Comment (Sriparno08) [inline content/plotly/concepts/express/terms/bar/bar.md:40]:
```suggestion
```py
```

File: content/plotly/concepts/express/terms/bar/bar.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+import plotly.express as px
+fig = px.bar(x=['a', 'b', 'c', 'd'], y=[1, 2, 3, 4], labels={'x':'Category', 'y':'Count'}, title='Simple Bar Chart', height=500, width=250)
+fig.show()
+```
+
+Output:
```
Comment (Sriparno08) [inline content/plotly/concepts/express/terms/bar/bar.md:46]:
```suggestion
The above code produces the following output:
```

File: content/plotly/concepts/express/terms/bar/bar.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+![Output of ploltly.express.bar() method example](https://raw.githubusercontent.com/Codecademy/docs/main/media/plotly-bar-example.png)
```
Comment (Sriparno08) [inline content/plotly/concepts/express/terms/bar/bar.md:48]:
```suggestion
![Output for the above example](https://raw.githubusercontent.com/Codecademy/docs/main/media/plotly-bar-example.png)
```

**Comment by M-MacNeill** [comment]
Thanks for the suggested changes @Sriparno08. I have now applied them.

**Review by Sriparno08 — APPROVED** [review]
Looks good for a second review! 👍

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @M-MacNeill! Thanks for contributing to Codecademy Docs!

I've suggested a few changes, could you please review and modify them at your earliest convenience?
Thank you!

**Comment by M-MacNeill** [comment]
@mamtawardhani, Thank you for the additional suggestions. I have now made them too.

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the required changes @M-MacNeill!

The entry looks good to me 👍
Will merge it to the main branch soon!

**Pull request merged**